octoprint_bettergrblsupport/__init__.py

```diff
@@ -319,11 +321,19 @@ def get_template_configs(self):
 
     # #-- EventHandlerPlugin mix-in
     def on_event(self, event, payload):
-        subscribed_events = Events.FILE_SELECTED + Events.PRINT_STARTED + Events.PRINT_CANCELLED + Events.PRINT_DONE + Events.PRINT_FAILED
+        subscribed_events = Events.FILE_SELECTED + Events.PRINT_STARTED + \
+                            Events.PRINT_CANCELLED + Events.PRINT_DONE + Events.PRINT_FAILED + \
+                            Events.CONNECTED
 
         if subscribed_events.find(event) == -1:
             return
 
+        if event == Events.CONNECTED:
```

Comment:
overall, I think I'm good with keeping track of the grbl version and doing version specific logic based on it